### ISSUE TITLE
Search engine seems worst than Documenter.jl

### ISSUE THREAD

**Issue by albertomercurio** [LuxDL/DocumenterVitepress.jl#190]
I have the impression that the search engine finds the results in a different way compared to the one of `Documenter.jl`. I find the search engine of the last one more efficient, while `DocumenterVitepress.jl` seems a bit confusing sometimes.

**Comment by lazarusA** [comment]
indeed. Some will need to pass the options from here https://github.com/JuliaDocs/Documenter.jl/blob/c955fe8112b5503ae363cc7d0e315eff10c37d63/assets/html/js/search.js#L189 into the config options here: https://github.com/LuxDL/DocumenterVitepress.jl/blob/56ad88c741bb1dea193f1dd7593be0df56ad07e3/docs/src/.vitepress/config.mts#L64

**Comment by albertomercurio** [comment]
Hello, any news about this? It would be great to have a search engine as powerful as the one in Documenter.jl

**Comment by albertomercurio** [comment]
Hello, any news on this?

**Comment by lazarusA** [comment]
no time. All my work on this package is for free and on me time. PRs are welcome though.

**Comment by lazarusA** [comment]
let's see if #384 is useful and could close this issue.

**Comment by albertomercurio** [comment]
I have made #385 with Codex 5.6 Sol, just to have an alternative solution. Codex says it is better than #384, but I don't know.
